geps/gep-1897/index.md

```diff
@@ -149,61 +117,53 @@ certificate, we implement a metaresource named `BackendTLSPolicy`, that was prev
 Because naming is hard, a new name may be
 substituted without blocking acceptance of the content of the API change.
 
-The selection of the applicable Gateway API persona is important in the design of BackendTLSPolicy, because it provides
-a way to explicitly describe the _expectations_ of the connection to the application.  BackendTLSPolicy is configured
-by the application developer Gateway API persona to signal what the application developer _expects_ in connections to
-the application, from a TLS perspective.  Only the application developer can know what the application expects, so it is
-important that this configuration be managed by that persona.
-
-During the course of discussion of this proposal, we did consider allowing the cluster operator persona to have some access
-to Gateway cert validation, but as mentioned, BackendTLSPolicy is used primarily to signal what the application
-developer expects in the connection.  Granting this expectation to any other role would blur the lines between role
-responsibilities, which compromises the role-oriented design principle of Gateway API. As mentioned in Non-goal #8,
-providing a mechanism for the cluster operator gateway role to override gateway to backend TLS settings is not covered
-by this proposal, but should be addressed in a future update.  One idea is to use two types: ApplicationBackendTLSPolicy,
-and GatewayBackendTLSPolicy, where the application developer is responsible for the former, the cluster operator is
-responsible for the latter, and the cluster operator may configure whether certain settings may be overridden by
-application developers.
-
-The BackendTLSPolicy must contain these configuration items to allow the Gateway to operate successfully
+A new [metaresource](../gep-713/index.md), `BackendTLSPolicy`, will be introduced.
+
+With TLS, there are, of course, two components involved: the client (the gateway implementation) and the server (an application, likely running in the cluster).
+The Gateway API cannot control the server behavior.
```

Comment:
I daresay:
```suggestion
The Gateway API cannot control the server behavior, so we've created the BackendTLSPolicy to signal what the application developer _expects_ in connections to
the server, from a TLS perspective.
```

Reply:
This is 100% an intended change of the semantics of the API to declare that the API declared "you should use TLS to connect with these properties". This is fundamentally different than the sentence above